pages/glossary/user-interaction.md

```diff
@@ -0,0 +1,11 @@
+---
+title: User interaction
+key: user-interaction
+unambiguous: true
+objective: true
+---
+
+A user interaction corresponds to any [DOM event][event] that has the property [`isTrusted`][isTrusted] set to `True`
```

Comment:
I don't think this is strictly true, `isTrusted` events can be fired from the browser too, such as onload.  Maybe just to make this more reusable we should add a list of events here?